src/components/Tabs/components/TabList/TabList.tsx

```diff
@@ -0,0 +1,69 @@
+import { KeyboardEvent, MutableRefObject } from "react"
```

Comment:
Might be worth adding stories for your TabList and TabListItem.

Reply:
Hmmm, I started down that path, but it felt like a bit much. Do you think it would be useful to, say, style those pieces in the abstract?

Reply:
I think stories or somehow breaking down the api a bit more in the docs. Maybe just some urls that point to respective components for changing the look of the tab items.

Reply:
Yeah, I think fleshing out the nested READMEs and linking to them from the main one would be helpful. Good callout!

Reply:
@thispastwinter take a look